### DISCUSSION TITLE
Title slot not returning strict Document library Title when query contentclass:STS_List_DocumentLibrary is used.

### DISCUSSION THREAD

**Discussion by forthekernel** [microsoft-search/pnp-modern-search#2070] [Q&A]
-- Applies to Search results 4.6.1 but this is also an issue for 4.5. --
I am having an issue were the default slot "Title" is returning an alternative string (ie: not the "Title" (name) of the document Library). 

Eg: Title from Pnp Search shows "Title": "NSW RES - Schubach St, 444, East Albury".
whereas the document library name is "Schubach St, 444, East Albury".

Can anybody tell me why this is may differ? 
(Afaik there have been no customizations applied to the managed properties for the sites site collection)

The unusual Title eg "NSW RES - *" seems to be set when creating a document library from a List template (.stp) but the library name is set correctly. There must be a property set when applying the list template that is causing this but I do not understand how or why.  How can I get the search webpart to display the actual NAME of the document Library?

**Comment by wobba** [comment]
Not sure why this is. Do you have multiple languages on your site? If so, can you check the Title value when you switch to the default language of the site? As each locale can show different names.

**Reply by forthekernel** [reply]
Hi Wobba, I have set the region settings for the site and have tried toggling the "use localization" webpart property with no success. I have determined that Sharepoint search gets the same Title "prefix" so is not related to the web parts.

**Reply by wobba** [reply]
I'm referring to default locale for a site and SharePoint support for having different labels on e.g. doc libs per locate. This has nothing to do with localization in the web part.
